spec/view.test.js

```diff
@@ -5,14 +5,21 @@ import { shallow } from 'enzyme';
 import { shallowToJson } from 'enzyme-to-json';
 import Photo from '../client/src/components/Photo.jsx';
 
-const photoUrl = 'https://cap-bnb-photo-service.s3.us-east-2.amazonaws.com/1-stream-belmont-3.jpg';
+const props = {
```

Comment:
I think all you need here is an object called `photo`. You seem to have nested this one layer too deep.